less/global.less

```diff
@@ -28,11 +32,18 @@ h3 {
 p {
   font-size: 1.6rem;
   line-height: 1.4;
+
+  @media @phone {
+      font-size: 3rem;
+  }
 }
 
 .container {
   max-width: 800px;
   width: 100%;
-  margin: 0 auto;  
-}
+  margin: 0 auto;
 
+  @media @phone {
+    width: 200%;
+  }
```

Comment:
you'll want this removed here, it is pushing everything out the side, i know it was a rush to the deadline because of the technical issues. i know you have a grasp of the material